app/views/parent/kids/edit.html.erb

```diff
@@ -1,6 +1,6 @@
-<h1>Create a Kid!</h1>
+<h1> Edit Kid Information </h1>
 
-<%= form_for @kid do |form| %>
+<%= form_for @kid, url: "/kids/#{params[:id]}" do |form| %>
```

Comment:
@TheTempas @Garo-A This is an instance of embedding post information into the code that we need to avoid. It was fine for when a director added/edited a counselor, but this is where we need to find a better solution. We can come up with a better solution in the future - this has been added to Trello as a bug: https://trello.com/c/ftX8FOW2/95-counselor-edit-page-dont-allow-modification-of-source-html-to-update-a-different-user